wd_core.au3

```diff
@@ -1229,7 +1229,10 @@ Func _WD_Startup()
 		Case @error = 0xDEAD And @extended = 0xBEEF
 			; function not available
 
-		Case @error
+		Case @error = $_WD_ERROR_GeneralError
```

Comment:
Can not edit this file thus can not propose changes in usuall way.

Why ?

btw. for this reason I posted code here:
https://github.com/Danp2/au3WebDriver/pull/457#issuecomment-1519846643

Reply:
Was there still an issue here? If not, then please mark resolved.

Reply:
Please consider to extend $sPortAvailable 

```autoit
		Case @error = $_WD_ERROR_GeneralError
			$sPortAvailable = " (Unknown error when tried to get free port)"

		Case @error
			$sPortAvailable = " (Unavailable free port)"

```

Reply:
> (Unavailable free port)

This would sound better as "Port unavailable". However, this is the current output -- 
`    _WD_Startup: Port:	445 (Unavailable)`

It already tells you that it is displaying information about the port so adding the word "port" again would be redundant IMO.

> (Unknown error when tried to get free port)

Similar comments here. Maybe "Unknown status", but remember that `_WD_GetFreePort` would have already written an error to the logs, such as -- 

`_WD_GetFreePort ==> Not found [13 / 445] : Parameters:   MinPort=445   MaxPort=Default > No available ports found`

I will add some comments to that section of code to better document it.

Reply:
So it looks fine.
No more comments, reuquests here.